src/adapter/src/error.rs

```diff
@@ -367,8 +367,13 @@ impl AdapterError {
                 "Try choosing one of the smaller sizes to start. Available sizes: {}",
                 expected.join(", ")
             )),
-            AdapterError::NoClusterReplicasAvailable(_) => {
-                Some("You can create cluster replicas using CREATE CLUSTER REPLICA".into())
+            AdapterError::NoClusterReplicasAvailable(_, is_managed) => {
+                Some(if *is_managed {
+                    "You can adjust the replication factor of the cluster with ALTER CLUSTER. \
+                    For example `ALTER CLUSTER <cluster-name> SET (REPLICATION FACTOR 1)`".into()
+                } else {
+                    "You can create cluster replicas using CREATE CLUSTER REPLICA".into()
```

Comment:
```suggestion
                    "Use ALTER CLUSTER to adjust the replication factor of the cluster. \
                    Example:`ALTER CLUSTER <cluster-name> SET (REPLICATION FACTOR 1)`".into()
                } else {
                    "Use CREATE CLUSTER REPLICA to attach clusters replicas to the cluster ".into()
```

Reply:
Feel free to ignore the suggestion if this disrupts the current style of hints, but this feels more direct than the alternative, using fewer words.